docs/usage/workload_storage.rst

```diff
@@ -0,0 +1,104 @@
+Workload Storage
+================
+
+Any workload needs different volumes with different storage capacities
+to host its components. These volumes are stored in :term:`LVM LV`.
+
+Volumes
+-------
+
+Following the total storage capacity needed, establish a number of
+volumes and their capacities for each node, as exemplified below:
+
+.. code-block:: yaml
+
+    metalk8s_lvm_lvs_vg_metalk8s:
+        lv01:
+            size: 52G
+        lv02:
+            size: 52G
+        lv03:
+            size: 52G
+        lv04:
+            size: 11G
+        lv05:
+            size: 11G
+        lv06:
+            size: 11G
+        lv07:
+            size: 5G
+        lv08:
+            size: 5G
+
+Resize LVs
+----------
+
+Volumes' capacities can be resized (one or several at once).
+Change the volume size value to a higher one and run:
+
+.. code::
+
+  ansible-playbook -i <inventory>/hosts -t storage playbooks/deploy.yml
+
+Logical volumes are automatically resized.
+
+Configuration layout
+--------------------
+
+The configuration can be applied to groups and hosts in two
+different ways.
+
+.. note::
+   Configuration files are merged for every created host or group.
+
+To apply a configuration, create a .yaml file in either (or both) group_vars
+and host_vars with the group name associated, or create a folder in group_vars
+or host_vars with several .yaml files.
+
+.. warning::
+   The ansible command above must be run at every file addition (deletions
+   are not allowed).
```

Comment:
What does this mean, 'deletions are not allowed'? Why couldn't you delete files from `group_vars` or `host_vars`? Of course you can.

I guess what's meant here is: LVs can't be removed. But that's a different context.